src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -587,8 +587,8 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
                         alreadyExists = true;
     
                         // Determine if order, action, active, async has changed
-                        if (defaultHandler.Load_Order__c != existingHandler.Load_Order__c || 
-                            defaultHandler.Trigger_Action__c != existingHandler.Trigger_Action__c ||
+                        if (defaultHandler.Load_Order__c != existingHandler.Load_Order__c ||
+                            !compareTriggerHandlerActions(defaultHandler, existingHandler) ||
```

Comment:
Should we also have a regression test for this?

Reply:
I was surprised there was no existing unit test for the trigger handler test specifically. One can be built though.

Reply:
If you wouldn't mind that'd be great.